resources/assets/js/translations/locales/fr.json

```diff
@@ -109,6 +108,24 @@
   "formValidation.tooLong": "Too long!",
   "formValidation.tooShort": "Too short!",
   "job.daysSinceClosed": "{dayCount, plural, =0 {Aucun jour} one {# jour} other {# jours}} depuis la fermeture",
+  "jobBuilder.culturePace.01.description": "Our deadlines are tight, we balance several tasks at the same time, and our priorities are always changing. Our work should come with running shoes!",
```

Comment:
Most of these look like translations for other files, they should be committed as part of another PR (or multiple PR's)

Reply:
That's a result of translations being added with out the `npm run translation` script being run. I will make a separate PR that is just updating the translations. That should fix this.

Reply:
Done. 👍